mage/query-modules/python/igraphalg.md

```diff
@@ -0,0 +1,258 @@
+
+#### Output:
+
+* `nodes: List[Vertex]` ➡   A list of nodes in topological sorted order.
+
+#### Usage:
+
+```cypher
+CALL igraphalg.topological_sort() YIELD *
+RETURN nodes;
+```
+
+### `maxflow(source, target, capacity)`
+The maximum flow problem consists of finding a flow through a graph such that it is the maximum possible flow.
+
+#### Input:
+
+* `source: Vertex` ➡ Source node from which the maximum flow is calculated
+* `target: Vertex` ➡ Sink node to which the max flow is calculated
+* `capacity: string (default="weight")` ➡ Edge property which is used as the flow capacity of the edge
+
+#### Output:
+
+* `max_flow: Number` ➡ Maximum flow of the network, from source to sink
+
+#### Usage:
+
+```cypher
+MATCH (source {id: 0}), (sink {id: 5})
+CALL igraphalg.maxflow(source, sink, "weight")
+YIELD max_flow
+RETURN max_flow;
+```
+
+### `mincut(source, target, capacity,directed)`
+Minimum cut calculates the minimum st-cut between two vertices in a graph.
+
+#### Input:
+
+* `source: Vertex` ➡ Source node from which the maximum flow is calculated
+* `target: Vertex` ➡ Sink node to which the max flow is calculated
+* `capacity: string (default="weight")` ➡ Edge property which is used as the capacity of the edge
+
+#### Output:
+
+* `node: Vertex` ➡ Vertex in graph.
+* `partition_id: int` ➡ Id of the partition where `node` belongs after min-cut.
+
+#### Usage:
+
+```cypher
+  MATCH (source {id: 0}), (sink {id: 5})
+  CALL igraphalg.mincut(source, sink)
+  YIELD node, partition_id 
+  RETURN node, partition_id;
+```
+
+### `community_leiden(objective_function, weights, resolution_parameter, beta, initial_membership, n_iterations, node_weights)`
+Finding community structure of a graph using the Leiden algorithm of Traag, van Eck & Waltman.
+
+#### Input:
+
+* `objective_function: string (default="CPM")` ➡ Whether to use the Constant Potts Model (CPM) or modularity. Must be either `CPM` or `modularity`.
+* `weights: string (default=NULL)` ➡ If a string is present, use this edge attribute as the edge weight if it isn't edge weights default to 1.
+* `resolution_parameter: float (default=1.0)` ➡ Higher resolutions lead to smaller communities, while lower resolutions lead to fewer larger communities.
+* `beta: float (default=0.01)` ➡ Parameter affecting the randomness in the Leiden algorithm. This affects only the refinement step of the algorithm.
+* `initial_membership: List[int](default=NULL)` ➡  If provided, the Leiden algorithm will try to improve this provided membership. If no argument is provided, the algorithm simply starts from the singleton partition.
+* `n_iterations: int (default=2)` ➡ The number of iterations to iterate the Leiden algorithm. Each iteration may improve the partition further.
+* `vertex_weights: List[float] (default=NULL) ➡ 	
+The vertex weights used in the Leiden algorithm. If this is not provided, it will be automatically determined based on the objective_function.
```

Comment:
```suggestion
* `vertex_weights: List[float] (default=NULL) ➡ The vertex weights used in the Leiden algorithm. If this is not provided, it will be automatically determined based on the objective_function.
```

Reply:
Ovdje je valjda falilo zatvaranje code formatinga, pa samo pliz provjeri jel sad ok